src/pybind/mgr/dashboard_v2/controllers/rgw.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+
+from ..tools import ApiController, RESTController
+from .. import logger
+
+
+@ApiController('rgw')
+class Rgw(RESTController):
+
+    def __init__(self):
+        self.daemon = RgwDaemon()
+
+
+@ApiController()
```

Comment:
Why did you not added the `path` argument to this decorator?

Reply:
Because it wouldn't be used. It seems like the `path` is only necessary for the initial controller attached, but not for all nested ones.

Reply:
I still don't understand what you are trying to accomplish with the code in this file.
You have two controller classes declared, and the second one is being instantiated in the constructor of the second one.

To be honest I don't even know what's the behavior of your code. Are you accessing the `RgwDaemon` controller using the `/api/rgw/daemon` URL? If that is the case, you don't need to decorate the `RgwDaemon` class with  `@ApiController`.